src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <a href="#" class="card__image-product">
+          <img
+            src="./images/imac.jpeg"
+            class="card__image"
+            alt="Product">
+        </a>
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina <br>
+          5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__discription">Product code: 195434</p>
+        <div class="card__stats">
```

Comment:
```suggestion
        <p class="card__discription">Product code: 195434</p>
     
        <div class="card__stats">
```
Add one empty line between sibling blocks inside the same parent block for both of them.
Same for all the similar cases above and below